code/modules/mining/mine_items.dm

```diff
@@ -74,6 +75,46 @@
 /obj/item/weapon/pickaxe/proc/playDigSound()
 	playsound(src, pick(digsound),50,1)
 
+/obj/item/weapon/pickaxe/capped //Used at the labor camp
+	name = "capped pickaxe"
+	desc = "A pickaxe capped with rubber softeners, considerably dampening its use a weapon. The caps themselves are fitted extremely tightlu."
+	icon_state = "pickaxe_capped"
+	force = 5 //At the end of the day, it's still a pickaxe
+	digspeed = 50 //Capped off. Realistically, it shouldn't be able to destroy things at all.
+	attack_verb = list("smashed", "slammed", "beaten")
+	var/uncapping = 0 //If the pickaxe is being manipulated
+
+/obj/item/weapon/pickaxe/capped/attack_self(mob/user) //Uncapping!
+	if(!ishuman(user))
+		user << "<span class='warning'>You don't have hands to manipulate [src]!</span>"

```

Comment:
Monkies. Also, the only other things that could be picking up a pickaxe and using it somehow are xenos, no?